index.html

```diff
@@ -0,0 +1,311 @@
+<!DOCTYPE html>
+
```

Comment:
- [ ] The provided Demo link of your project is not working. Kindly host your project on Github pages and update the demo link accordingly